pkg/apis/ptp/v1/ptpoperatorconfig_types.go

```diff
@@ -14,6 +14,7 @@ type PtpOperatorConfigSpec struct {
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
 	DaemonNodeSelector map[string]string `json:"daemonNodeSelector"`
+	EventConfig PtpEventConfig `json:"ptpEventConfig,omitempty"`
```

Comment:
I think you need to remove the omitempty here

Reply:
```
apiVersion: ptp.openshift.io/v1
kind: PtpOperatorConfig
metadata:
  name: default
spec:
  daemonNodeSelector:
    feature.node.kubernetes.io/ptp-capable: "yes" 
  #ptpEventConfig: {}
```